distribution/docs/src/main/jdocs/content/_configuring/environment-hardening-contents.adoc

```diff
@@ -23,8 +23,7 @@ It is recommended to apply the following security mitigations to the ${branding}
 
 |JMX
 |tampering, information disclosure, and unauthorized access
-a|* Remove `-Dcom.sun.management.jmxremote` from `<${branding}_HOME>/bin/karaf`. +
-* Disable ${branding}'s JMX management `rmiRegistryPort` and `rmiServerPort` (`1099`, `44444`) by removing these entries from `etc/org.apache.karaf.management.cfg`. +
+a|* Disable ${branding}'s JMX management `rmiRegistryPort` and `rmiServerPort` (`1099`, `44444`) by removing these entries from `etc/org.apache.karaf.management.cfg`. +
```

Comment:
❗️ It probably makes more sense to shut the RMI ports by default in order to prevent JMX connections from the local host as well.

Please see the section on [*Local Monitoring and Management* here](http://docs.oracle.com/javase/7/docs/technotes/guides/management/agent.html) for more information about local vs. remote JMX connectivity. Removing this property turns off remote management but does not prevent local-host JMX connections to the running Karaf instance.